### ISSUE TITLE
Provide git lfs in the VM

### ISSUE THREAD

**Issue by obilodeau** [Varying-Vagrant-Vagrants/VVV#1569]
## Expected Behavior

A git clone from a git lfs repo works.

## Current Behavior

Files stored in LFS will not be properly checked-out. It will be text files containing refs.

## Possible Solution

Add these steps to the provisioning phase:

```
curl -s https://packagecloud.io/install/repositories/github/git-lfs/script.deb.sh | sudo bash
apt install git-lfs
git lfs install
```

## Context

Web repositories usually contain lots of images. Git LFS simplifies and accelerates git repositories that contain a lot of binary content. It is offered by GitHub and self-hosting is easy. We use it for our custom site used with vvv.

## Your Environment

* VVV version: 2.2.1
* VVV Git Branch: master

**Comment by tomjn** [comment]
I've a strong distaste of pulling in a script like that via curl during provisioning, at WCEU we had long delays getting machines up and running because we couldn't download vagrant hosts updater over the wifi, which is only 10kb

Since we know half of the dependencies the script checks for are already installed during provisioning, and we know in advance what the OS is, and already add several GPG keys to apt, the script shouldn't be needed.

So, if you'd like this in VVV core:

 - Add `git-lfs` to the package list in `provision/provision.sh`
 - Download the signing key if necessary and put it in the `config/apt-keys`
 - Modify the `package_install` function to check if it needs adding and add it
 - make it run `git lfs install`

For now though, you can either:

 - Add it as a utility, and put the steps in your issue inside a `provision.sh` file e.g. a git repo named git-lfs, containing `git-lfs/provision.sh` then added to `vvv-custom.yml`
 - Manually do it inside the VM

I'm going to mark this as a feature request in the meantime

**Comment by tomjn** [comment]
An additional option would be to checkout the LFS parts using `git` on the host machine

**Comment by obilodeau** [comment]
Thanks for the great feedback! Right now I've solved it by modifying the `vvv-init.sh` script out of our fork of [Varying-Vagrant-Vagrants/custom-site-template](https://github.com/Varying-Vagrant-Vagrants/custom-site-template). Since I feel you are willing to accept a PR that would solve it properly, I'm going to try to go ahead and do that.

> An additional option would be to checkout the LFS parts using git on the host machine, I imagine a git fetch && git checkout would probably be what's needed, or something similar, I'm not super familiar with LFS

LFS (Large Files Support) fetches the large files via an out-of-band HTTPS server. You can't `git fetch` or `git checkout` your way out of this, unfortunately.

**Comment by obilodeau** [comment]
PR is in: #1570

**Comment by tomjn** [comment]
Closing this, Git LFS support is included in v2.3 of VVV thanks to the mentioned PR
